src/main/java/com/github/manolo8/darkbot/core/BotInstaller.java

```diff
@@ -93,7 +93,7 @@ private boolean tryInstall() {
         BotInstaller.SEP = API.readMemoryInt(mainApplicationAddress.get() + 4);
 
         if ((query = API.queryMemory(bytesToSettings, 1)).length != 1) return true;
-        this.settingsAddress.send(query[0] - 237);
+        this.settingsAddress.send(query[0] - 277);
```

Comment:
Is this change not going to break anything else (eg: in 2d mode)?

Reply:
Just moved settings address class offset (-40) everywhere, now it points exactly to `Settings` class instance